src/main/scala/cromwell/util/google/GoogleCredentialFactory.scala

```diff
@@ -91,11 +92,12 @@ class UserCredentialFactory(userId: String, secretsPath: Path, dataStoreDir: Pat
   }
 
   protected def initCredential(): Credential = {
-    val clientSecrets = GoogleClientSecrets.load(jsonFactory, new FileReader(secretsPath.toFile))
-    val flow = new GoogleAuthorizationCodeFlow.Builder(httpTransport, jsonFactory, clientSecrets, scopes)
+    val clientSecrets = GoogleClientSecrets.load(GoogleCredentialFactory.jsonFactory,
+      new FileReader(secretsPath.toFile))
+    val builder = new GoogleAuthorizationCodeFlow.Builder(GoogleCredentialFactory.httpTransport,
+      GoogleCredentialFactory.jsonFactory, clientSecrets, GoogleCredentialFactory.scopes)
       .setDataStoreFactory(new FileDataStoreFactory(dataStoreDir.toFile))
-      .build()
-    new AuthorizationCodeInstalledApp(flow, new GooglePromptReceiver).authorize(userId)
+    new AuthorizationCodeInstalledApp(builder.build(), new GooglePromptReceiver).authorize(userId)
```

Comment:
No real reason. Once I use the aforementioned DRY import, I may switch it back. The line wrapping to 120 chars was just getting annoying, and this was the last one I ended up with.